attributes/default.rb

```diff
@@ -40,3 +40,18 @@
   DES-CBC3-SHA
   !DSS
 )
+
+override['nrpe']['packages'] = %w(
+  nrpe
+  nrpe-selinux
+  nagios-plugins
+  nagios-plugins-disk
+  nagios-plugins-dummy
+  nagios-plugins-load
+  nagios-plugins-mailq
+  nagios-plugins-nrpe
+  nagios-plugins-ntp
+  nagios-plugins-procs
+  nagios-plugins-swap
+  nagios-plugins-users
+)
```

Comment:
Why is this being added here? This shouldn't be needed and taken care of by osl-nrpe.